package.json

```diff
@@ -2,8 +2,8 @@
   "name": "wappuapp-backend",
   "version": "1.0.0",
   "engines": {
-    "node": "5.7.x",
-    "npm": "2.x.x"
+    "node": "^9",
+    "npm": "^5.6.x"
```

Comment:
Aren't we using yarn, not npm?

Reply:
Rojektiluarnio used yarn and Hommasoft used npm, could probably go with either one. Any preferences?

Reply:
I've recently had quite a lot of headaches with NPM5 package locking, heard ~good things about yarn. Maybe just stick to yarn for now, see how it goes?

Reply:
+1 vote for yarn from me as well. No particular reason as to why, just haven't had major issues with it thus far.

Reply:
Alright!